javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -403,7 +403,17 @@
 	/**
 	 * <a href='https://www.datadog.com/'>Datadog</a> host for accessing API. 'api.datadoghq.com' is default.
 	 */
-	DATADOG_API_HOST("datadog-api-host");
+	DATADOG_API_HOST("datadog-api-host"),
+
+	/**
+	 * Integer value as minimum time collector for current requests
+	 */
+	CURRENT_REQUEST_MIN_DURATION_LOG("current-request-min-duration"),
+
+	/**
+	 * Integer value as minimum time collector for current requests
+	 */
+	CURRENT_REQUEST_STACKTRACE_SIZE("current-request-stacktrace-size");

```

Comment:
What would be a typical value for this size? In which case would a different value than the typical value would be absolutely needed?